### ISSUE TITLE
Quest suggestion: elevator dimensions

### ISSUE THREAD

**Issue by derhuerst** [streetcomplete/StreetComplete#5932]
### General
Affected tag(s) to be modified/added: [Elevator dimensions](https://wiki.openstreetmap.org/wiki/Proposal:Elevator_dimensions)
- `height` & `maxheight:physical`
- `length` & `maxlength:physical`
- `width` & `maxwidth:physical`
- `door:width` & `door:height`

Questions asked:
- **What is the {height,width,length} of this elevator?**
- **What is the clear {height,width,length} of this elevator that is not obstructed by a barrier (e.g. a lamp)?**
- **What is the {height,width} of the elevator's door? Pick the smaller door if there are multiple**

### Checklist
Checklist for quest suggestions (see [guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md)):
- [x] 🚧 To be added tag is established and has a useful purpose
- [x] 🤔 Any answer the user can give must have an equivalent tagging (Quest should not reappear to other users when solved by one) – I think so...
- [ ] 🐿️ Easily answerable by any pedestrian from the outside but a survey is necessary – I don't understand the question. It is not answerable easily, as you need a tool to measure things.
- [x] 💤 Not an overwhelming percentage of quests have the same answer (No spam)
- [x] 🕓 Applies to a reasonable number of map data (Worth the effort) – I checked Germany to get an idea, it has ~10k elevators.

**Comment by matkoniecz** [comment]
> I don't understand the question

StreetComplete should not include things easily surveyable remotely (therefore "a survey is necessary") and should have questions that are reasonable person walking around, without entering non-public spaces ("Easily answerable by any pedestrian from the outside")

Do you have suggestion for better phrasing here?

**Comment by derhuerst** [comment]
> StreetComplete should not include things easily surveyable remotely (therefore "a survey is necessary") and should have questions that are reasonable person walking around, without entering non-public spaces ("Easily answerable by any pedestrian from the outside")
> 
> Do you have suggestion for better phrasing here?

What about "Is it something that cannot be answered remotely, where people have to be right next to it? Also, is it easily answerable by someone out and about, without being an expert on the matter?"

**Comment by mnalis** [comment]
**TL;DR:** perhaps only ask for elevator door maximum usable width (or even simpler, for wheelchair usability)?

---

Do note that [Quest guidelines](https://github.com/streetcomplete/StreetComplete/blob/master/QUEST_GUIDELINES.md) suggest (among other things):

> ⚛️ Atomic quests: Per quest, only one thing should need to be answered by the user.

And this suggestion seems to want 8 separate answers, so that would be **8 separate quests**. 

Also, while I can see how the dimensions of the elevator can be theoretically[^1] useful in same cases, that is rather weak _"🤷 Useful purpose"_ and probably fails _"🕓 Effort vs impact"_ / _"🐿️ Easy answer"_.


One common idea that might actually be supported (and is in reasonable demand) that I can think of is **wheelchair routing**, but that would care mostly **only about usable door width**, but not the other 7 sub-quests (and perhaps even that could be reduced to quicker and more wildly supported `wheelchair=yes/no` quest)

[^1]: i.e. I don't see much _practical_ use. Is there some data consumer which handles those? E.g. a router where you can enter dimensions of your bicycle and if it can be rotated in 3D (e.g. MTB might, but cargo bikes not), and it makes a best route using elevators in which the bike would fit in some orientation (ie horizontal, or vertical, or in diagonal etc)? 
Or some other use? Because I doubt people will e.g. plan their route by manually checking those... (e.g. although I personally would like to know if my touring bike would fit in some elevator on railway station without unpacking the panniers and the rest of the load; I doubt if I'd invest the time to manually analyze the data, especially given that a negative-fit answer would only lead to me being annoyed earlier and thus for longer, without doing much to helping in majority of the cases :smiley_cat:)

**Comment by westnordost** [comment]
I agree with @mnalis , thank you for the write-up.

Also, additionally, I would say that a quest that asks for either `wheelchair=yes/no` or `maxwidth` on elevators would also not fulfill the quest guidelines because 
- **💤 No spam**: Elevators in public spaces are usually *made **for*** wheelchair users, it would be very very rare that these are not accessible by them. [Paternosters](https://en.wikipedia.org/wiki/Paternoster_lift) are super-rare and (too) slim elevators are also rare and usually only [retro-fitted in old apartment buildings](https://www.youtube.com/shorts/IF7l6C_fesA), i.e. are usually not mapped on OSM either.
- **🐿️ Easy answer**: Users don't know how wide a wheelchair is. So there'd be no way around measuring.
